dispatchr-ruby/app/models/request_item.rb

```diff
@@ -1,4 +1,9 @@
 class RequestItem < ApplicationRecord
   belongs_to :request, optional: true
   belongs_to :item, optional: true
+
+  validates :request_id, presence: true
+  validates :item_id, presence: false
```

Comment:
when you create a new item with request_item it creates a new itemid for you. so unless you know the item id of the item you want, its not mecessary